docs/docs/concepts/services-api.mdx

```diff
@@ -0,0 +1,13 @@
+---
+id: services-api
+title: Services & APIs
+---
+
+The Ory Cloud exposes APIs to its services to easily integrate your applications using our SDKs or
+HTTP Endpoints directly. The Ory Cloud APIs make it possible for you to integrate your own user interface (UI)
+in the form of a Web or Native Application.
```

Comment:
Is this possible now?
If not I would either omit it or refer to the GitHub issue/roadmap/relevant doc.